osu.Game/Beatmaps/Formats/LegacyBeatmapEncoder.cs

```diff
@@ -220,8 +220,15 @@ LegacyControlPointProperties getLegacyControlPointProperties(ControlPointGroup g
                 var samplePoint = legacyControlPoints.SamplePointAt(group.Time);
                 var effectPoint = legacyControlPoints.EffectPointAt(group.Time);
 
+                // if samplePoint isn't already legacy, create LegacyHitSampleInfo with customSampleBank 1
```

Comment:
If we're going to go with this, I want much better inline documentation describing what the expectations are, and what `1` means.